File: app/src/main/java/com/andraganoid/verymuchtodo/ListsFragment.java
package com.andraganoid.verymuchtodo;


import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class ListsFragment extends Fragment {


    public ListsFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View flView = inflater.inflate(R.layout.fragment_lists, container, false);


        return flView;
    }

}

File: app/src/main/java/com/andraganoid/verymuchtodo/ktodo/lists/ListsFragment.kt
package com.andraganoid.verymuchtodo.ktodo.lists

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.lifecycle.ViewModelProviders
import com.andraganoid.verymuchtodo.R
import com.andraganoid.verymuchtodo.ktodo.TodoBaseFragment

class ListsFragment : TodoBaseFragment() {


    private lateinit var viewModel: ListsViewModel

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.lists_fragment, container, false)
    }

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        viewModel = ViewModelProviders.of(this).get(ListsViewModel::class.java)
        // TODO: Use the ViewModel
    }

}
